terraform/custom_domains/environment_domains/variables.tf

```diff
@@ -0,0 +1,32 @@
+variable "zone" {
+  type        = string
+  description = "Name of DNS zone"
+}
+variable "front_door_name" {
+  type        = string
+  description = "Name of Azure Front Door"
+}
+variable "resource_group_name" {
+  type        = string
+  description = "Name of resouce group name"
+}
+variable "domains" {
+  description = "List of domains record names"
+}
+variable "environment_tag" {
+  type        = string
+  description = "Environment"
+} # added from tra
```

Comment:
I'm not quite sure what this comment means here?

Reply:
removed